src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -0,0 +1,289 @@
+import $ from 'jquery';
```

Comment:
can you remove https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/group/externalIssues.jsx? I think it should be unused given these changes: https://github.com/getsentry/sentry/pull/9058/files#diff-fc551d814b9396a399a42e67c9c5ff1cL315